presto-main/src/main/java/com/facebook/presto/operator/aggregation/DoubleSumAggregation.java

```diff
@@ -42,10 +47,10 @@ public static void combine(@AggregationState NullableDoubleState state, @Aggrega
         if (state.isNull()) {
             state.setNull(false);
             state.setDouble(otherState.getDouble());
-            return;
         }
-
-        state.setDouble(state.getDouble() + otherState.getDouble());
+        else {
```

Comment:
Is there a performance difference between this and original code?

Reply:
Done